benchmark.py

```diff
@@ -28,15 +29,25 @@
 from utils.bazel import Bazel
 from utils.output_handling import export_csv, upload_csv
 
+# TMP has different values, depending on the platform.
+TMP = tempfile.gettempdir()
+
+def _platform_path_str(posix_path):
+  """Converts the path to the appropriate format for platform."""
+  if os.name == 'nt':
+    return posix_path.replace('/', '\\')
```

Comment:
is there no python built-in library that does that for you?

Reply:
As discussed offline, there's `pathlib` but it's not built-in for python 2.